packer/plugin.go

```diff
@@ -65,7 +66,7 @@ func (c *Config) Discover() error {
 		}
 	}
 
-	// Next, look in the default plugins directory inside the configdir/.packer.d/plugins.
+	// Next, look in the default plugins directory inside the configdir/.d/plugins.
```

Comment:
why are we changing this location?

Reply:
woops, that's a mistake !
```suggestion
	// Next, look in the default plugins directory inside the configdir/.packer.d/plugins.
```